pkg/network/ddnpmapi.h

```diff
@@ -122,12 +122,13 @@ typedef struct _filterAddress
     uint8_t                   v4_address[4];    // address in network byte order, so v4_address[0] = top network tuple
     uint8_t                   v4_padding[4];    // pad out to 64 bit boundary
     uint8_t                   v6_address[16];
-    uint64_t                  mask; // number of mask bits.  
+    uint64_t                  mask; // number of mask bits.
 } FILTER_ADDRESS;
 
 #define     DIRECTION_INBOUND    ((uint64_t)0)
 #define     DIRECTION_OUTBOUND   ((uint64_t)1)
 
+//#define     FILTER_LAYER_IPPACKET       ((uint64_t) 0)
```

Comment:
Is there a reason this is commented out/should it be removed in general?

Reply:
I copy/pasted from the driver to make sure it was identical.

Reply:
https://github.com/DataDog/datadog-windows-filter/blob/d0c8ded189ec4dafb28612311d13aa63948b3425/include/ddnpmapi.h#L131

Reply:
Makes sense! LGTM to me then.